www/source/images/solutions-logos/icon-habitat.svg

```diff
@@ -1,6 +1,12 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 50 50">
```

Comment:
Same here, we should just remove this, and its reference also (looks like there's only one, in the Node tutorial).